rollouts/user-manuals/api-specifications.md

```diff
@@ -1,14 +1,65 @@
 # API Specifications
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 A basic example for Kruise Rollouts resource YAML:
 
+**Note: v1beta1 available from Kruise Rollout v0.5.0.**
+
+<Tabs>
+  <TabItem value="v1beta1" label="v1beta1" default>
+
 ```yaml
 apiVersion: rollouts.kruise.io/v1alpha1
 kind: Rollout
 metadata:
   name: rollouts-demo
   # The rollout resource needs to be in the same namespace as the corresponding workload
-  namespace: defaults
+  namespace: default
+spec:
+  # rollout of published workloads, currently only supports Deployment, CloneSet, StatefulSet, Advanced StatefulSet
+  workloadRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: echoserver
+  strategy:
+    canary:
+      steps:
+      ### the 1-st batch ###
+      # routing 5% traffics to the new version
+      - traffic: "5%"
+        # Need Manual confirmation before enter to next batch
+        pause: {}
+        # optional, The first step of released replicas. If not set, the default is to use 'weight', as shown above is 5%.
+        replicas: 1
+      ### the 2-nd batch ###
+      - traffic: "50%"
```

Comment:
it seems not necessary to add quotes around traffic value, so can we remove the quotes so that the format of traffic and replicas can be the same?